games.yaml

```diff
@@ -1839,6 +1940,21 @@
         - image: http://freesiege.sourceforge.net/screens/screen0.png
         - image: http://freesiege.sourceforge.net/screens/screen1.png
 
+- name: Siege of Avalon
+  clones:
+    name: Siege of Avalon Open Source
```

Comment:
You forgot to prepend name with `-` here. :) Unfortunately YAML is a bit brittle. I tried to improve error reporting a bit.